Dockerfile.cdot

```diff
@@ -0,0 +1,19 @@
+# Build container
```

Comment:
Is this different from the Dockerfile? Do we need it?

Reply:
This was not different and unnecessary, so it has been removed.